From b583ce85417befdc3d8a13458ed9b81e9421b665 Mon Sep 17 00:00:00 2001
From: Babanpreet Singh <bbnpreetsingh@gmail.com>
Date: Thu, 30 Jul 2026 06:06:17 +0000
Subject: [PATCH 12963/15333] watchdog: booke_wdt: Document unused parameter of
 __booke_wdt_disable()

scripts/kernel-doc -none reports:

  Warning: drivers/watchdog/booke_wdt.c:158 function parameter 'data'
  not described in '__booke_wdt_disable'

__booke_wdt_disable() is invoked through on_each_cpu(), so it takes a
void * argument that it does not use. Its kernel-doc comment does not
describe the parameter at all.

Document it as unused.

Assisted-by: Claude:claude-opus-5
Signed-off-by: Babanpreet Singh <bbnpreetsingh@gmail.com>
Link: https://lore.kernel.org/r/20260730060617.7-3-bbnpreetsingh@gmail.com
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 drivers/watchdog/booke_wdt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/watchdog/booke_wdt.c b/drivers/watchdog/booke_wdt.c
index 932a03f4436a..1b1fb389b89b 100644
--- a/drivers/watchdog/booke_wdt.c
+++ b/drivers/watchdog/booke_wdt.c
@@ -149,6 +149,7 @@ static void __booke_wdt_enable(void *data)
 
 /**
  * __booke_wdt_disable - disable the watchdog on the given CPU
+ * @data: unused
  *
  * This function is called on each CPU.  It disables the watchdog on that CPU.
  *
-- 
2.55.0

